src/modules/settings/index.js

```diff
@@ -150,16 +155,10 @@ class SettingsModule {
 
         const changelogPanel = $('#bttvChangelog .tse-content');
         api.get('changelog').then(data => {
-            if (data.status === 200) {
-                changelogPanel.append('<h1>Changelog</h1');
-                data.changelog.forEach(entry => {
-                    const pubdate = window.moment(entry.publishedAt).format('MMM D, YYYY');
-                    changelogPanel.append(`<h2>Version ${entry.version} (${pubdate})</h2> ${entry.body}`);
-                });
-            } else {
-                debug.log(`Changelog bad status: ${data.status}`);
-                changelogPanel.append('<h2>Error fetching changelog.</h2>');
-            }
+            data.changelog.forEach(entry => {
+                const pubdate = window.moment(entry.publishedAt).format('MMM D, YYYY');
```

Comment:
date should be computed within the template, and `pubdate` is not camelcase.. i would also prefer just `date` or the original key name.